src/main/webapp/styles/general.css

```diff
@@ -4,6 +4,10 @@
   --backColor: white;
   --mainColor: #4285f4;
   --placeholder: #b7b7b7;
+  --fGray: #f0f0ec;
+  --gRed: #db4437;
```

Comment:
what does the "f" "g" and "gL" mean?

Reply:
i made the var gRed bc its the google red

Reply:
f is for forum